client/src/app/components/AppDialogShell.vue

```diff
@@ -10,11 +10,13 @@
         div(class="dialog-body")
           slot(name="body") default body
         div(class="dialog-footer")
-          div(class="dialog-footer-errors")
+          div(class="dialog-footer-errors is-pulled-left")
             div(v-show="errors.length")
-              p {{ errorDirections }}
-              ul
-                li(v-for="error in errors") {{ error }}
+              span {{ errorDirections }} 
+              span(v-for="(error, index) in errors", class="error-color")
+                span(v-if="errors.length > 0 && index < errors.length - 1") {{ error }}, 
+                span(v-else) {{ error }}
```

Comment:
What are these two lines (17,18) doing?  I think they can be removed and {{ error }} placed at the end of line 16

Reply:
@bryan-gilbert I have added this in order to add a comma to all the errors, except for the last one. Since I have implemented an inline logic for every one of them, I think it's a good way to separate them visually. I can remove it, if needed. Let me know what your thoughts on it are. 

However, upon revisiting it, I think that this is not a good way of approaching it. So I will pushing another commit shortly replacing it with `errors.join(', ')` which will properly separate them. 

Thank you for your comment!